proxy/src/config.rs

```diff
@@ -299,23 +302,35 @@ impl<'a> TryFrom<&'a Strings> for Config {
         let tls_settings = match (tls_trust_anchors?,
                                   tls_end_entity_cert?,
                                   tls_private_key?,
-                                  pod_name?.as_ref())
+                                  tls_pod_identity_template?.as_ref(),
+                                  tls_controller_identity?)
```

Comment:
TIOLI: this seems like enough stuff to match on that maybe it ought to be a `struct` of its own?

Reply:
I think it would be reasonable to factor it out as its own struct but there cons as well as pros. For example, if it were a struct and we changed this match's arms so that they didn't match every field of the struct, we might accidentally skip a field, e.g. skip reporting an error when a sometimes-required field is missing.